modules/contact/src/actions/ContactAction.C

```diff
@@ -368,72 +401,157 @@ ContactAction::addMortarContact()
       _problem->addVariable(var_type, variable_name, params);
     };
 
-    // Normal contact: same family/order as primal.
+    // To avoid instabilities in the solution and obtain the full benefits of a variational
+    // enforcement, use of dual mortar is strongly recommended.
```

Comment:
This comment about dual mortar seems like it's kind of floating in space, not really close to any lines of code that do anything with dual

Reply:
I intended to put it before we create the Lagrange's multipliers, where you have other relevant comments. I moved it to the description of the `_use_dual` parameters, which might make sense while we have it.